opendatahub-io/data-science-pipelines-operator
# feat: add the ability to configure artifact download link expiry

## The issue resolved by this Pull Request:
Part of https://issues.redhat.com/browse/RHOAIENG-8225

Depends on:  https://github.com/opendatahub-io/data-science-pipelines/pull/62

## Description of your changes:

This allows a user to configure the expiry time (in seconds) of an artifact download link via the DSPA. Default is 15 seconds.

## Testing instructions

1. Deploy DSPO: Use this one https://github.com/opendatahub-io/data-science-pipelines-operator/pull/670
2. Deploy DSPA with `.spec.apiServer.artifactSignedURLExpirySeconds` set to some value integer.
3. Confirm this value is set in the dsp server deployment as: `SIGNED_URL_EXPIRY_TIME_SECONDS`



## Checklist
